src/module/actor/data-model-character.js

```diff
@@ -73,6 +73,12 @@ export default class OseDataModelCharacter extends foundry.abstract.DataModel {
       encumbrance: new SchemaField({
         value: new NumberField({ integer: false }),
         max: new NumberField({ integer: false }),
+        pct: new NumberField({ integer: false }),
+        steps: new ObjectField(),
+        atHalfEncumbered: new BooleanField(),
+        atQuarterEncumbered: new BooleanField(),
+        atThreeEighthsEncumbered: new BooleanField(),
+        color: new StringField(),
```

Comment:
I'm *pretty sure* the schema is for what's persisted to the DB.  All we persist, as far as I remember, is `max`.

TBH `value` already shouldn't be in there, as it's computed based on the items you're carrying; all we really give the ability to modify is `max`.